internal/client/client.go

```diff
@@ -310,7 +310,7 @@ func (c *CSAPI) DoFunc(t *testing.T, method string, paths []string, opts ...Requ
 	}
 	// debug log the request
 	if c.Debug {
-		t.Logf("Making %s request to %s", method, reqURL)
+		t.Logf("Making %s request to %s", method, req.URL)
```

Comment:
This adds the query parameters to the logged URL

Reply:
Thanks! :D

Reply:
Split out to https://github.com/matrix-org/complement/pull/250